src/aether-elasticsearch-consumer/templates/deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         release: {{ .Release.Name }}
     spec:
       containers:
-        - name: consumer
+        - name: es-consumer
```

Comment:
@MrPink Can we do this here? If so, does it change how it's referenced in the `gateway-manager/services.yml`?